src/App.js

```diff
@@ -1,7 +1,9 @@
+import FaqIcon from "./components/atoms/FaqIcon";
```

Comment:
take this line out. it is just for your local testing

Reply:
remove

Reply:
Done